src/functions-and-arrays.js

```diff
@@ -24,15 +53,38 @@ function sum() {}
 
 // Iteration #4: Calculate the average
 // Level 1: Array of numbers
+
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
-function averageNumbers() {}
+function averageNumbers(numbersAvg) {
+
+let sum = 0;
+if(!numbersAvg.length) return null;
+for (let i=0;i<numbersAvg.length;i++) {
+  sum += numbersAvg[i];
+}
+return sum / numbersAvg.length;
+}
+
+console.log(averageNumbers(numbersAvg));
+
 
 
 // Level 2: Array of strings
 const wordsArr = ['seat', 'correspond', 'linen', 'motif', 'hole', 'smell', 'smart', 'chaos', 'fuel', 'palace'];
 
-function averageWordLength() { }
+function averageWordLength(arr) { 
+let sum = 0;
```

Comment:
Same here, all your code inside the function should be written in relation with the function parameter, like this:
```
function averageWordLength(arr) { 
let sum = 0;
if (!arr.length) return null;

for (let i=0; i<arr.length; i++) {
  sum += arr[i].length;
 }
return sum / arr.length;
}
```